enterprise/server/remote_execution/snaploader/snaploader.go

```diff
@@ -392,9 +392,30 @@ func (l *FileCacheLoader) CacheSnapshot(ctx context.Context, key *fcpb.SnapshotK
 		ar.OutputFiles = append(ar.OutputFiles, out)
 		eg.Go(func() error {
 			ctx := egCtx
-			d, err := fileDigest(filePath)
-			if err != nil {
-				return err
+			var d *repb.Digest
+			if *snaputil.EnableLocalSnapshotSharing || *snaputil.EnableRemoteSnapshotSharing {
```

Comment:
uber nit: maybe slightly more direct to check `key.RunnerId`

```suggestion
			if key.GetRunnerId() == "" {
```

Reply:
Hopefully this gets cleaned up soon anyway, but I like the flags because the intention is really obvious